batch/batch/worker/worker.py

```diff
@@ -677,21 +677,15 @@ def __init__(self, batch_id, user, gsa_key, job_spec, format_version):
 
         req_cpu_in_mcpu = parse_cpu_in_mcpu(job_spec['resources']['cpu'])
         req_memory_in_bytes = parse_memory_in_bytes(job_spec['resources']['memory'])
-        if 'storage' in job_spec['resources']:
-            req_storage_in_bytes = parse_memory_in_bytes(job_spec['resources']['storage'])
```

Comment:
Just want to confirm this was due to Ben's bug and new job specs no longer have a `storage` key?

Reply:
The opposite. All job_specs should have a "resources" dictionary with "storage" in it now.

Reply:
Got it, thanks!